sass/_secondary.scss

```diff
@@ -0,0 +1,19 @@
+.sidebar__section {
+  display: none;
+  @media screen and (min-width: 800px) {
+    display: block;
+    max-width: 200px;
+    margin: 10px auto;
+    padding-top: 10px;
+  > h1 {
+    font-size: .8em;
+    font-weight: bold;
+    color: #106DB3;
+  }
+  > p {
+    font-size: .7em;
+    line-height: 20px;
+    color: lighten( $text_dark, 20%);
+  }
```

Comment:
`> h1` & `> p` and their definitions aren't indented far enough, so it appears as though they are on a level they are not